examples/android/app/src/main/jni/main.cpp

```diff
@@ -86,13 +86,13 @@ int main(int argc, char* argv[])
     sf::RenderWindow window(screen, "");
     window.setFramerateLimit(30);
 
-    const auto texture = sf::Texture::loadFromFile("image.png").value();
+    const auto texture = sf::Texture::createFromFile("image.png").value();
```

Comment:
If we're going to have throwing constructors then I'd much rather we use those throwing constructors than call `std::optional::value`.